VideoAPI/VideoApi.Domain/Conference.cs

```diff
@@ -175,5 +177,15 @@ public Participant GetVideoHearingOfficer()
         {
             return Participants.SingleOrDefault(x => x.IsVideoHearingOfficer());
         }
+
+        public IList<Message> GetMessages()
+        {
+            return Messages.OrderByDescending(x => x.TimeStamp).ToList();
+        }
+        public void AddMessage(string from, string to, string messageText)
```

Comment:
I can't seem to find unit tests for these methods. Look under the Domain => Conference folder in Unit Tests project for examples.

'AddParticipantTests'

Reply:
Done